docs/data/base/components/popup/popup.md

```diff
@@ -0,0 +1,80 @@
+---
+productId: base-ui
+title: React Popup component
+components: Popup
+githubLabel: 'component: popup'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/
+---
+
+# Popup
+
+<p class="description">The Popup component is a utility that lets you display content in tooltips and popovers.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+{{"component": "modules/components/ComponentPageTabs.js"}}
+
+## Introduction
+
+The Popper is a utility component for creating various kinds of popups.
+It relies on the [Floating UI](https://floating-ui.com/) third-party library for positioning.
+
+:::info
+Base UI also contains the [Popper](/base-ui/react-popper/) component with a very similar API.
+It's based on the [Popper](https://popper.js.org/) library, which is in maintenance mode and it's deprecated itself.
```

Comment:
```suggestion
It's based on the [Popper](https://popper.js.org/) library which, itself, is deprecated and in maintenance mode.
```

Reply:
I meant that the Popper component is deprecated here. I guess there's a way to frame this better.

How about:

> Base UI also contains the deprecated [Popper](/base-ui/react-popper/) component with a very similar API.
> It's based on the [Popper](https://popper.js.org/) library, which is in maintenance mode.